src/css/bootstrap-overrides.scss

```diff
@@ -1,6 +1,11 @@
 @import "variable-overrides";
 @import "../../node_modules/bootstrap/scss/bootstrap";
 
+#root {
+  width: 100%;
+  position: absolute;
```

Comment:
Why? Wouldn't relative work if you just need it for other absolutes?

Reply:
ah! I did that because the page changed width when the dialog was shown!
look: 

![screencapture-localhost-3000-2019-04-04-08_51_05](https://user-images.githubusercontent.com/1106587/55557038-f8388100-56b6-11e9-9754-4ffa30debbb7.png)

Reply:
Oh wow. Ok then, if absolute fixes it then it's fine. (maybe make sure that the Sorter menu still works)